BuildResidentialHPXML/measure.rb

```diff
@@ -0,0 +1,3573 @@
+    arg.setDefaultValue(1)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeStringArgument("heat_pump_backup_heating_capacity", true)
+    arg.setDisplayName("Heat Pump: Backup Heating Capacity")
+    arg.setDescription("The backup output heating capacity of the heat pump. If using '#{Constants.SizingAuto}', the autosizing algorithm will use ACCA Manual S to set the capacity.")
+    arg.setUnits("Btu/hr")
+    arg.setDefaultValue(Constants.SizingAuto)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument("mini_split_is_ducted", true)
+    arg.setDisplayName("Mini-Split: Is Ducted")
+    arg.setDescription("Whether the mini-split heat pump is ducted or not.")
+    arg.setDefaultValue(false)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument("evap_cooler_is_ducted", true)
+    arg.setDisplayName("Evaporative Cooler: Is Ducted")
+    arg.setDescription("Whether the evaporative cooler is ducted or not.")
+    arg.setDefaultValue(false)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument("heating_system_flow_rate", true)
+    arg.setDisplayName("Heating System: Flow Rate")
+    arg.setDescription("The flow rate of the heating system.")
+    arg.setUnits("CFM")
+    arg.setDefaultValue(0)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument("cooling_system_flow_rate", true)
+    arg.setDisplayName("Cooling System: Flow Rate")
+    arg.setDescription("The flow rate of the cooling system.")
+    arg.setUnits("CFM")
+    arg.setDefaultValue(0)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument("hvac_distribution_system_type_dse", true)
+    arg.setDisplayName("HVAC Distribution: Uses Distibution System Efficiency")
+    arg.setDescription("Whether the HVAC distribution system type is distribution system efficiency.")
+    arg.setDefaultValue(false)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument("annual_heating_dse", true)
+    arg.setDisplayName("HVAC Distribution: Annual Heating Distribution System Efficiency")
+    arg.setDescription("The annual heating efficiency of the distribution system.")
+    arg.setDefaultValue(0.8)
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument("annual_cooling_dse", true)
+    arg.setDisplayName("HVAC Distribution: Annual Cooling Distribution System Efficiency")
+    arg.setDescription("The annual cooling efficiency of the distribution system.")
+    arg.setDefaultValue(0.7)
+    args << arg
+
+    hvac_control_type_choices = OpenStudio::StringVector.new
+    hvac_control_type_choices << "manual thermostat"
+    hvac_control_type_choices << "programmable thermostat"
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument("hvac_control_type", hvac_control_type_choices, true)
```

Comment:
Remove this argument, doesn't affect the model.

Reply:
OK.